<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="btn_choose_picture">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Choose picture</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_file_path">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>70</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_flip_vtc">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertically</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_flip_hrz">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>70</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizontally</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_flip_both">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>70</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertically and Horizontally</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Image Flipping</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_image">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>501</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_edited_image">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>501</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Before</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>480</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>After</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_export_folder">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_choose_folder">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>540</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose folder</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_filename_export">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>580</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>File name export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_export">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>580</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
